app/Phragile/TransactionLoader.php

```diff
@@ -0,0 +1,31 @@
+<?php
+namespace Phragile;
+
+class TransactionLoader {
+	private $transactionFilter;
+
+	public function __construct(TransactionFilter $transactionFilter)
+	{
+		$this->transactionFilter = $transactionFilter;
+	}
+
+	/***
+	 * Loads task transactions from Phabricator in batches of 200
+	 *
+	 * @param $taskIDs
+	 * @param PhabricatorAPI $phabricatorAPI
+	 * @return array
+	 */
+	public function load($taskIDs, PhabricatorAPI $phabricatorAPI)
```

Comment:
I'd say the PhabricatorAPI object is a service that should be passed in via the constructor.